UnitTests/GlobalFixture.cs

```diff
@@ -16,11 +20,15 @@ public class GlobalFixture : IDisposable
     {
         public readonly ServiceProvider _serviceProvider;
         public readonly IAuthorizationService _authorizationService;
-
+        public readonly UserManager<ApplicationUser> _userManager;
+        public readonly string _testDataDirectory = "test_data/automatically_deleted/";
```

Comment:
The tests are failing because the project has a dependency on the DATA_DIRECTORY ending in "/data" (see here: https://github.com/classtranscribe/WebAPI/blob/8d8cedcdb5f348277992e9e3c2d4f0f2f6a1facc/ClassTranscribeServer/Startup.cs#L179)

Can you make the `_testDataDirectory` variable in `GlobalFixture.cs` to end in "/data"? Such as:
```
public readonly string _testDataDirectory = "test_data/automatically_deleted/data/";
```